docs/content/preview/migrate/performance.md

```diff
@@ -70,6 +70,8 @@ These performance optimizations apply whether you are importing data using the y
 
 By default, yb-voyager exports four tables at a time. To improve data export, parallelize the export of data from multiple tables using the [–-parallel-jobs](../reference/yb-voyager-cli/#parallel-jobs) argument with the export data command to increase the number of jobs. Setting the value too high can however negatively impact performance; a setting of '4' typically performs well.
 
+If you use EXPERIMENTAL_FAST_EXPORT to [accelerate data export](../migrate-steps/#accelerate-data-export-for-mysql-or-oracle), `--parallel-jobs` will not have any impact on performance and yb-voyager will export only one table at a time.
```

Comment:
**Note to the developers:**
@sanyamsinghal @makalaaneesh shall we log a WARNING also in the code for this - if this arg is used and the accelerated export is attempted then just show a warning on the screen.

Reply:
yeah i think it can be helpful also , in case user is not aware of the env var is set and assuming normal export.